backfila-embedded/build.gradle.kts

```diff
@@ -18,12 +27,8 @@ dependencies {
   testImplementation(Dependencies.kotlinTest)
 }
 
-val jar by tasks.getting(Jar::class) {
-  archiveBaseName.set("backfila-embedded")
```

Comment:
was this `archiveBaseName.set("backfila-embedded")` doing anything?

Reply:
probably only useful if you’re also using ShadowJar